t/core/utils.t

```diff
@@ -239,3 +236,32 @@ res:tell David to
 res:John and David
 res: and David
 res:John and \$me
+
+
+
+=== TEST 7:  resolve with search
+--- config
+    location /t {
+        content_by_lua_block {
+            local core = require("apisix.core")
+            local resolvers = {"223.5.5.5", "223.6.6.6"}
+            local search = {"google.com"}
+            local ndots = 5
+            core.utils.init_dns_proxy({
+                ndots = ndots,
+                search = search,
+                resolvers = resolvers,
+            })
+            local ip_info, err = core.utils.dns_parse("news.google.com")
```

Comment:
Try to resolve domain like `news`, since we have the custom `search` option, `news.google.com` will be resolved.

Also, it's not good to depend on external suite, it might cause the CI unstable.

Reply:
> Try to resolve domain like `news`, since we have the custom `search` option, `news.google.com` will be resolved.
> 
> Also, it's not good to depend on external suite, it might cause the CI unstable.

It should the dns serve support, such as core dns, we can not mock it. We can change it to github.com to avoid the CI unstable.

Reply:
I try to use test with https://github.com/Kong/lua-resty-dns-client/blob/master/spec/client_spec.lua#L219 , but the method "_search_iter"  is not export.So I use the special domain to do the test.

Reply:
Then how to proof the code can be used correctly in k8s pod? Say we need to resolve the etcd service with its short name like "myetcd", while actually the record in dns is "myetcd.ns.svc.cluster.local" as far as I know.

Reply:
> Then how to proof the code can be used correctly in k8s pod? Say we need to resolve the etcd service with its short name like "myetcd", while actually the record in dns is "myetcd.ns.svc.cluster.local" as far as I know.

It is my fault, It can work now.